p_sets/p_set_1.rb

```diff
@@ -51,5 +51,41 @@ def mario
   pyramid_printer calculate_row_length ask_height
 end
 
+#TIMEFORCHANGE
+#assume that your program’s input will be 9.00 or just 9 but, again, not $9 or 900.
+
+COINS = [100,20,10,5,2,1]
+
+def ask_change
+  while true
+    puts "How much change is owed?"
+    change = gets.chomp.to_f
+    break if change > 0
+  end
+   return change
+end
+
+def calculate_coins(pence)
+  n = COINS.count
+  coins = []
+  for n in 0..n-1
+    denomination = COINS[n]
```

Comment:
Because `n` only gets used to access the coin in the array, you could replace this with 
```
COINS.each do |denomination|
...
end
```
which would seem more ruby-ish.